EDU-10771 Fix JSON Error (#766)
vtexdocs/dev-portal-content: docs/guides/Integration-Guides/tax-service-integration-guide/tax-services-specification.md

In synchronous integration, VTEX’s Checkout API triggers and sends a request to the external tax service API whenever there are changes to a customer’s cart, such as adding or removing items.

> >⚠️ Timeout for the request is five seconds.\n- There is no retry in case of timeout.\n- If the external service that responds to the request times out constantly, the store will not be able to finish the order.\n- If this integration is active, it applies to all stores in that account.

Reviewer: ```suggestion
>⚠️ Timeout for the request is five seconds. There is no retry in case of timeout. If the external service that responds to the request times out constantly, the store will not be able to finish the order. If this integration is active, it applies to all stores in that account.
```

The `authorizationHeader` defines the value that the Checkout will use in the `Authorization` header of calls to the external tax calculation API. This field can be used to define the access credentials for this API.

Once the POST for the Checkout Configuration API has finished processing the request with this data, its synchronous integration with the Tax API is activated.

> > ℹ When a purchase is made in a store, the location from which the order is shipped matters for tax calculation purposes. Because of this, when items from [White Label Sellers](https://help.vtex.com/pt/tutorial/definicoes-de-conta-franquia-e-seller-white-label--5orlGHyDHGAYciQ64oEgKa#) are part of an order, tax configuration for the marketplace (seller 1) is not taken into account for those items. Each seller must have its own tax service configuration in order for this type of integration function properly.

Reviewer: ```suggestion
>⚠️ When a purchase is made in a store, the location from which the order is shipped matters for tax calculation purposes. Because of this, when items from [White Label Sellers](https://help.vtex.com/en/tutorial/white-label-seller--5orlGHyDHGAYciQ64oEgKa) are part of an order, tax configuration for the marketplace (seller 1) is not taken into account for those items. Each seller must have its own tax service configuration in order for this type of integration function properly.
```

This body has eight main fields:

> | Field               | Type   | Description                                                                                                                                                                         |
> |---------------------|--------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

Reviewer: ```suggestion
| Field | Type | Description |
| - | - | - |
```

>ℹ️ For the Checkout API to understand the request body, the content-type must be set to `application/vnd.vtex.checkout.minicart.v1+json`.

#### Jurisdiction fields

If you use Avalara as your tax calculation provider, response bodies might also include the following fields, which refer to the different jurisdictions that may apply according to location.

| Field       | Type   | Description                                               |
|-------------|--------|-----------------------------------------------------------|
| `jurisType` | string | Type of jurisdiction that applies to calculation.         |
| `jurisCode` | string | Unique code that identifies the appropriate jurisdiction. |
| `jurisName` | string | Name of the Jurisdiction that applies to the calculation. |

These fields are also read by Checkout and added to the `priceTag`.

Below is an example for values that may be contained in these fields, and you can download all of the jurisdictions and respective codes used by Avalara [here](https://help.avalara.com/Avalara_AvaTax_Update/Download_jurisdiction_codes).

```json
{
  "jurisType": "State",
  "jurisCode": "20",
  "jurisName": "Kansas"
}
```

[block:html]
{

>   "html": "<link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css\" integrity=\"sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm\" crossorigin=\"anonymous\">\n<script src=\"https://code.jquery.com/jquery-3.2.1.slim.min.js\" integrity=\"sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN\" crossorigin=\"anonymous\"></script>\n<script src=\"https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js\" integrity=\"sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q\" crossorigin=\"anonymous\"></script>\n<script src=\"https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js\" integrity=\"sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl\" crossorigin=\"anonymous\"></script>\n\n\n<a href=\"tax-service-integration-guide\"<button type=\"button\" class=\"btn btn-outline-secondary\">Back</button></a>\n\n<style></style>"

Author: Não possível realizar a correção sugerida, pois o trecho mencionado se trata de um HTML, e está correto. Uma vez inserida a correção, o HTML ficará inválido, pois a correção é específica em markdown.
https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md

This body has eight main fields:

- `orderFormId`: *string* related to the order form ID.

- `salesChannel`: type of sales channel.

- `items`: an *array* that contains objects which are the order products, where **dockId** is a field that refers to its identification on the logistics system that contains information of its address.

- `totals`: an *array* with the total amount of the order form, divided into taxes, shipping, discounts, and the items themselves.

> - `clientEmail`: *string* that contains the client's email.

Reviewer: ```suggestion
- `clientEmail`: Client's email in string format.
```

> - `shippingDestination`: *object* with shipping information, it's a mandatory field.

Reviewer: ```suggestion
- `shippingDestination`: Object with shipping information. Mandatory field.
```

> - `shippingDestination`: *object* with shipping information, it's a mandatory field.

Reviewer: Só esse campo é obrigatório? E os outros?

Author: Pelo que eu li sobre a documentação, acredito que somente esse campo é obrigatório.

> - `clientData`: *object* that contains information regarding the client that did the order.

Reviewer: ```suggestion
- `clientData`: Object that contains information regarding the client that made the order.
```

> - `paymentData`: *object* that contains an *array* of payments, where there is information regarding the payment methods, etc.

Reviewer: ```suggestion
- `paymentData`: Object that contains an array of payments, where there is information regarding the payment methods, etc.
```

In response to the request sent by Checkout, we expect an array of products, each with its own array of taxes. See the example below:

```json
[
    {
        "id": "0",
        "taxes": [
            {
                "name": "TAX 1",
                "description": "",
                "value": 3.48
            },
            {
                "name": "TAX 2",
                "description": "",
                "value": 22
            }
        ]
    }
]
```

> - `id`: is the request item index, which means the SKU position on the items array sent by the requisition body.

Reviewer: ```suggestion
- `id`: Request item index, which means the SKU position on the items array sent by the requisition body.
```

> - `taxes`: is an array that contains all the taxes types for an SKU.

Reviewer: ```suggestion
- `taxes`: Array that contains all the taxes types for an SKU.
```

> - `name`: is the tax name that will appear on the checkout.

Reviewer: ```suggestion
- `name`: Tax name that will appear on the checkout.
```

- `description`: is an informative field. It will not appear on the storefront.

> - `value`: is the absolute numeric value that will be added to the original price.

Reviewer: ```suggestion
- `value`: Absolute numeric value that will be added to the original price.
```

In the example above, the only item in the items array has a cost of `10`, and, including the calculated taxes returned by the tax calculation tool, the total value would be `10 + 3.48 + 22 = 35.48`.

>ℹ️ If no taxes apply to the items in the order, the expected response is an empty array (`[]`).

> >ℹ️ For the Checkout API to understand the request body, the content-type must be set to `application/vnd.vtex.checkout.minicart.v1+json`

Reviewer: ```suggestion
>ℹ️ For the Checkout API to understand the request body, the content-type must be set to `application/vnd.vtex.checkout.minicart.v1+json`.
```

In response to the request sent by Checkout, we expect an array of products, each with its own array of taxes. See the example below:

```json
[
    {
        "id": "0",
        "taxes": [
            {
                "name": "TAX 1",
                "description": "",
                "value": 3.48
            },
            {
                "name": "TAX 2",
                "description": "",
                "value": 22
            }
        ]
    }
]
```

- `id` is the request item index, which means the SKU position on the items array sent by the requisition body;

> - `taxes` is an array that contains all the taxes types for an SKU;

Reviewer: Percebi que algumas listas não numeradas aparecem com `:`, outras não. Alguns tópicos terminam com `;` e outros não.

O nosso padrão é usar `:` e encerrar cada tópico com `.`, como é feito na seção **Jurisdiction fields**. Acho que vale padronizar todas as listas do guia dessa forma.

Outra opção que acho boa é transformar em uma tabela, com as colunas `Field`, `Type` (string, boolean, number, integer...) e `Description`.

| `taxes`       | array  | Is an array that contains all the taxes types for an SKU.                                                |
| `name`        | string | Is the tax name that will appear on the checkout.                                                        |
| `description` | string | Is an informative field. It will not appear on the storefront.                                           |
| `value`       | number | Is the absolute numeric value that will be added to the original price.                                  |

In the example above, the only item in the items array has a cost of `10`, and, including the calculated taxes returned by the tax calculation tool, the total value would be `10 + 3.48 + 22 = 35.48`.

>ℹ️ If no taxes apply to the items in the order, the expected response is an empty array (`[]`).

>ℹ️ For the Checkout API to understand the request body, the content-type must be set to `application/vnd.vtex.checkout.minicart.v1+json`.

#### Jurisdiction fields

If you use Avalara as your tax calculation provider, response bodies might also include the following fields, which refer to the different jurisdictions that may apply according to location.

> | Field       | Type   | Description                                               |
> |-------------|--------|-----------------------------------------------------------|
> | `jurisType` | string | Type of jurisdiction that applies to calculation.         |
> | `jurisCode` | string | Unique code that identifies the appropriate jurisdiction. |
> | `jurisName` | string | Name of the Jurisdiction that applies to the calculation. |

Reviewer: ```suggestion
| Field | Type | Description |
| - | - | - |
| `jurisType` | string | Type of jurisdiction that applies to calculation. |
| `jurisCode` | string | Unique code that identifies the appropriate jurisdiction. |
| `jurisName` | string | Name of the jurisdiction that applies to the calculation. |
```

> - `value` is the absolute value that will be added to the original price;

Reviewer: Importante mencionar que o value deve ser do tipo [number](https://json-schema.org/understanding-json-schema/reference/numeric#number).

In the example above, the only item in the items array has a cost of `10`, and, including the calculated taxes returned by the tax calculation tool, the total value would be `10 + 3.48 + 22 = 35.48`.

> [block:callout]

Reviewer: Corrigir a formatação dos callouts para `> ℹ`
